motionplan/motionPlanner.go

```diff
@@ -248,7 +248,7 @@ type planner struct {
 }
 
 func newPlanner(frame frame.Frame, seed *rand.Rand, logger logging.Logger, opt *plannerOptions) (*planner, error) {
-	solver, err := ik.CreateCombinedIKSolver(frame, logger, opt.NumThreads, opt.GoalThreshold)
+	solver, err := ik.CreateCombinedIKFrameSolver(frame, logger, opt.NumThreads, opt.GoalThreshold)
```

Comment:
[q]: RSDK-8852 will remove `frame` as an argument here and instead pass in the full framesystem?

Reply:
Nope, this is just a new helper function. 8852 will create a combined IK solver without using a frame, via a different function.